### ISSUE TITLE
Info logging break flutter hot reload

### ISSUE THREAD

**Issue by rebaz94** [lulupointu/vrouter#169]
Hello
I really like your package and starting to use in my project after more than one hour trying to fix why hot reload not working in the IDE. and tried everything to fix but nothing changed until I change logging of  VRouter to ```VLogs.warning``` and everything work perfectly.

I don't know why this happened only in the IDE but it work in terminal

**Comment by lulupointu** [comment]
Oh that's weird.

Which IDE do you use ?

**Comment by rebaz94** [comment]
I am using Android Studio

```
Android Studio 4.1.2
Build #AI-201.8743.12.41.7042882, built on December 20, 2020
Runtime version: 1.8.0_242-release-1644-b3-6915495 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.4
GC: ParNew, ConcurrentMarkSweep
Memory: 3966M
Cores: 4
Registry: documentation.show.toolbar=true, ide.new.welcome.screen.force=true, external.system.auto.import.disabled=true
Non-Bundled Plugins: Statistic, co.anbora.labs.firebase-syntax-highlighting, com.thoughtworks.gauge, org.intellij.plugins.markdown, com.intellij.marketplace, org.jetbrains.kotlin, pl.pszklarska.pubversionchecker, Dart, io.flutter, com.localizely.flutter-intl
```

And flutter 
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 2.6.0-12.0.pre.959, on Mac OS X 10.15.4 19E287 darwin-x64, locale en)
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.
[!] Xcode - develop for iOS and macOS (Xcode 12.4)
    ! Flutter recommends a minimum Xcode version of 13.
      Download the latest version or update via the Mac App Store.
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Community Edition (version 2021.1.3)
[✓] VS Code (version 1.62.3)
[✓] Connected device (2 available)

! Doctor found issues in 2 categories.
```

**Comment by lulupointu** [comment]
I've just tried using the [default example](https://pub.dev/packages/vrouter/example) and everything works as expected (using android studio, linux version though)

Do you have this issue with the default example or is it specific to your project?

Also did you tried `flutter clean` and `Invalid cache & restart` (in android studio) to see if that solves the issue ?

**Comment by rebaz94** [comment]
Sorry for the delay
I have issue only in my project, its very weird, created new project with same structure that cause the problem and it working without problem but when I copied same code below to my project it break hot reload until I remove logging.

also tried cleaning & even restarting system does not working


<details>
<summary>Here the code</summary>

```dart
void main() {
  // same thing with or without riverpod
  // runApp(
  //   const ProviderScope(
  //     child: MyApp(),
  //   ),
  // );
  runApp(
    const MyApp(),
  );
}

class MyApp extends StatefulWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  State<MyApp> createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> {
  @override
  Widget build(BuildContext context) {
    return VRouter(
      //logs: VLogs.warning,
      debugShowCheckedModeBanner: false,
      title: 'Test',
      theme: ThemeData.dark().copyWith(
        primaryColor: Colors.green,
        iconTheme: Theme.of(context).iconTheme.copyWith(color: Colors.white),
      ),
      themeMode: ThemeMode.dark,
      builder: (context, child) {
        return Shortcuts(
          manager: LoggingShortcutManager(),
          shortcuts: const {
            SingleActivator(LogicalKeyboardKey.escape): CancelFocusedIntent(),
          },
          child: child,
        );
      },
      initialUrl: '/some/123/nested/456/route1',
      routes: [
        VWidget(
          path: '/login',
          widget: const LoginScreen(),
        ),
        VGuard(
          beforeEnter: (vRedirector) async => !LoginScreen.isLoggedIn ? vRedirector.to('/login') : null,
          stackedRoutes: [
            VNester(
              path: '/some/:id/nested/:anotherId',
              widgetBuilder: (child) => Builder(
                builder: (context) {
                  return Scaffold(
                    body: child,
                  );
                },
              ),
              nestedRoutes: [
                VWidget(
                  path: 'route1',
                  name: 'route1',
                  widget: Builder(
                    builder: (context) {
                      return Scaffold(
                        body: Center(
                          child: Column(
                            mainAxisSize: MainAxisSize.min,
                            children: [
                              Text('Route1 screen'),
                              TextButton(
                                onPressed: () => context.vRouter.toNamed(
                                  'route2',
                                  pathParameters: {
                                    'id': '123',
                                    'anotherId': '456',
                                  },
                                ),
                                child: Text('go to route 2'),
                              )
                            ],
                          ),
                        ),
                      );
                    },
                  ),
                  transitionDuration: const Duration(seconds: 2),
                  buildTransition: (animation, secondaryAnimation, child) {
                    return FadeTransition(
                      opacity: animation,
                      child: child,
                    );
                  },
                ),
                VWidget(
                  path: 'route2',
                  name: 'route2',
                  widget: Builder(
                      builder: (context) {
                        return Container(
                          color: Colors.red,
                          child: TextButton(
                            onPressed: () => context.vRouter.toNamed(
                              'route1',
                              pathParameters: {
                                'id': '123',
                                'anotherId': '456',
                              },
                            ),
                            child: Text('go to route1'),
                          ),
                        );
                      }
                  ),
                  transitionDuration: const Duration(seconds: 1),
                  buildTransition: (animation, secondaryAnimation, child) {
                    return FadeTransition(
                      opacity: animation,
                      child: child,
                    );
                  },
                )
              ],
            ),
          ],
        ),
        VRouteRedirector(
          path: '/',
          redirectTo: '/some/123/nested/456/route1',
        ),
        VWidget(
          path: '*',
          widget: Scaffold(
            body: Container(
              color: Colors.red,
              child: Builder(
                builder: (context) {
                  final path = context.vRouter.path;
                  return Center(
                    child: Text('Page Not founded for: $path'),
                  );
                },
              ),
            ),
          ),
        ),
      ],
    );
  }
}

class LoginScreen extends StatelessWidget {
  const LoginScreen({Key? key}) : super(key: key);

  static bool isLoggedIn = false;

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: TextButton(
          onPressed: () {
            isLoggedIn = true;
            VRouter.of(context).to('/');
          },
          child: const Text('Press to Login'),
        ),
      ),
    );
  }
}

class LoggingShortcutManager extends ShortcutManager {
  @override
  KeyEventResult handleKeypress(BuildContext context, RawKeyEvent event) {
    final KeyEventResult result = super.handleKeypress(context, event);
    if (result == KeyEventResult.handled) {
      print('Handled shortcut $event in $context');
    }
    return result;
  }
}

class CancelFocusedIntent extends Intent {
  const CancelFocusedIntent();
}

``` 
</details>


<details>
<summary>Logs</summary>

```
Performing hot restart...
Syncing files to device macOS...
Restarted application in 1,393ms.
flutter: [VRouter: INFO] Stopped the navigation to /some/123/nested/456/route1 which used VRouter.to o
flutter: [VRouter: INFO] Successfully navigated to "/login" using VRouter.to o
[{"id":35,"result":{"value":"macOS","type":"_extensionType","method":"ext.flutter.platformOverride"}}]
[{"id":37,"result":{"timeDilation":"1.0","type":"_extensionType","method":"ext.flutter.timeDilation"}}]
[{"id":38,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaint"}}]
[{"id":39,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaintBaselinesEnabled"}}]
[{"id":40,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.repaintRainbow"}}]
[{"id":41,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.showPerformanceOverlay"}}]
[{"id":42,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.debugAllowBanner"}}]
[{"id":36,"result":{"value":"macOS","type":"_extensionType","method":"ext.flutter.platformOverride"}}]
[{"id":43,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.debugAllowBanner"}}]
[{"id":44,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":45,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":46,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.show"}}]
[{"id":47,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":48,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
[{"id":49,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":50,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":51,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966-c077c7f6ed5d","id":"10","progressId":"hot.reload","message":"Performing hot reload...","finished":false}}]
[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966-c077c7f6ed5d","id":"11","progressId":null,"message":"Syncing files to device macOS...","finished":false}}]
[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966-c077c7f6ed5d","id":"11","progressId":null,"finished":true}}]
[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966-c077c7f6ed5d","id":"10","progressId":"hot.reload","finished":true}}]
Reloaded 1 of 1395 libraries in 564ms.
[{"id":52,"result":{"code":0,"message":"Reloaded 1 of 1395 libraries"}}]

```

</details>

**Comment by rebaz94** [comment]
Also the these kind of logs never seen before, is it from VRouter?
```
[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966-c077c7f6ed5d","id":"10","progressId":"hot.reload","message":"Performing hot reload...","finished":false}}]
```

**Comment by lulupointu** [comment]
> `[{"event":"app.progress","params":{"appId":"af7f7828-af64-4ba1-8966c077c7f6ed5d","id":"10","progressId":"hot.reload","message":"Performing hot reload...","finished":false}}]`

I don't think those kin of logs are produced by VRouter.

I tried the code you gave in android studio but have no issue with hot restart. Unfortunately I don't think I could solve the issue with the limited information I have. If that's ok let's close this since you can solve if by using `logs: VLogs.warning`. If someone else raises the information and more information allows my to solve it I will try to come back here and ping you.

**Comment by rebaz94** [comment]
I will open again anytime I can reproduce outside of my project.
Thanks for helping

**Comment by yk-sgr** [comment]
I have the same issue. Setting the Loglevel to VLog.warning fixed it.

**Comment by lulupointu** [comment]
@yk-sgr can you share the output of `flutter doctor` ?

And which IDE do you use when this bug happens ?

**Comment by yk-sgr** [comment]
![](https://i.imgur.com/zwJcmoh.png)

Android Studio

**Comment by lulupointu** [comment]
For which platform are you developing ?

@rebaz94 can you also answer this one ?

**Comment by yk-sgr** [comment]
Android (12)

**Comment by lulupointu** [comment]
Is it reproducible with any project using VRouter or only one? For example have you an issue with the [default example](https://pub.dev/packages/vrouter/example)

**Comment by yk-sgr** [comment]
It's my first project with VRouter. Issues started after even doing somethink simple like this:
```dart
VRouter(
      routes: [
            VWidget(path: '/', widget: ProfileScreen()),
            VWidget(path: '/settings', widget: SettingsWidget()),
      ],
)
```
https://vrouter.dev/guide/Essentials/Getting%20Started

**Comment by lulupointu** [comment]
Can you run the example bellow and tell me if hot reload is broken ?

<details>
<summary>
Click to show code
</summary>

```dart
import 'package:flutter/material.dart';
import 'package:vrouter/vrouter.dart';

void main() {
  runApp(
    VRouter(
      debugShowCheckedModeBanner: false, // VRouter acts as a MaterialApp
      mode: VRouterMode.history, // Remove the '#' from the url
      logs: VLogs.info, // Defines which logs to show, info is the default
      routes: [
        VWidget(
          path: '/login',
          widget: LoginWidget(),
          stackedRoutes: [
            ConnectedRoutes(), // Custom VRouteElement
          ],
        ),
        // This redirect every unknown routes to /login
        VRouteRedirector(
          redirectTo: '/login',
          path: r'*',
        ),
      ],
    ),
  );
}

// Extend VRouteElementBuilder to create your own VRouteElement
class ConnectedRoutes extends VRouteElementBuilder {
  static final String profile = 'profile';

  static void toProfile(BuildContext context, String username) =>
      context.vRouter.to('/$username/$profile');

  static final String settings = 'settings';

  static void toSettings(BuildContext context, String username) =>
      context.vRouter.to('/$username/$settings');

  @override
  List<VRouteElement> buildRoutes() {
    return [
      VNester.builder(
        // .builder constructor gives you easy access to VRouter data
        path:
            '/:username', // :username is a path parameter and can be any value
        widgetBuilder: (_, state, child) => MyScaffold(
          child,
          currentIndex: state.names.contains(profile) ? 0 : 1,
        ),
        nestedRoutes: [
          VWidget(
            path: profile,
            name: profile,
            widget: ProfileWidget(),
          ),
          VWidget(
            path: settings,
            name: settings,
            widget: SettingsWidget(),

            // Custom transition
            buildTransition: (animation, ___, child) {
              return ScaleTransition(
                scale: animation,
                child: child,
              );
            },
          ),
        ],
      ),
    ];
  }
}

class LoginWidget extends StatefulWidget {
  @override
  _LoginWidgetState createState() => _LoginWidgetState();
}

class _LoginWidgetState extends State<LoginWidget> {
  String name = 'bob';
  final _formKey = GlobalKey<FormState>();

  @override
  Widget build(BuildContext context) {
    return Material(
      child: Center(
        child: Column(
          mainAxisSize: MainAxisSize.min,
          children: [
            Row(
              mainAxisSize: MainAxisSize.min,
              crossAxisAlignment: CrossAxisAlignment.center,
              children: [
                Text('Enter your name to connect: '),
                Container(
                  width: 200,
                  decoration: BoxDecoration(
                    border: Border.all(color: Colors.black),
                  ),
                  child: Form(
                    key: _formKey,
                    child: TextFormField(
                      textAlign: TextAlign.center,
                      onChanged: (value) => name = value,
                      initialValue: 'bob',
                    ),
                  ),
                ),
              ],
            ),
            SizedBox(
              height: 20,
            ),

            // This FAB is shared and shows hero animations working with no issues
            FloatingActionButton(
              heroTag: 'FAB',
              onPressed: () {
                setState(() => (_formKey.currentState!.validate())
                    ? ConnectedRoutes.toProfile(context, name)
                    : null);
              },
              child: Icon(Icons.login),
            )
          ],
        ),
      ),
    );
  }
}

class MyScaffold extends StatelessWidget {
  final Widget child;
  final int currentIndex;

  const MyScaffold(this.child, {required this.currentIndex});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('You are connected'),
      ),
      bottomNavigationBar: BottomNavigationBar(
        currentIndex: currentIndex,
        items: [
          BottomNavigationBarItem(
              icon: Icon(Icons.person_outline), label: 'Profile'),
          BottomNavigationBarItem(
              icon: Icon(Icons.info_outline), label: 'Info'),
        ],
        onTap: (int index) {
          // We can access this username via the path parameters
          final username = VRouter.of(context).pathParameters['username']!;
          if (index == 0) {
            ConnectedRoutes.toProfile(context, username);
          } else {
            ConnectedRoutes.toSettings(context, username);
          }
        },
      ),
      body: child,

      // This FAB is shared with login and shows hero animations working with no issues
      floatingActionButton: FloatingActionButton(
        heroTag: 'FAB',
        onPressed: () => VRouter.of(context).to('/login'),
        child: Icon(Icons.logout),
      ),
    );
  }
}

class ProfileWidget extends StatefulWidget {
  @override
  _ProfileWidgetState createState() => _ProfileWidgetState();
}

class _ProfileWidgetState extends State<ProfileWidget> {
  int count = 0;

  @override
  Widget build(BuildContext context) {
    // VNavigationGuard allows you to react to navigation events locally
    return VWidgetGuard(
      // When entering or updating the route, we try to get the count from the local history state
      // This history state will be NOT null if the user presses the back button for example
      afterEnter: (context, __, ___) => getCountFromState(context),
      afterUpdate: (context, __, ___) => getCountFromState(context),
      child: Padding(
        padding: const EdgeInsets.symmetric(horizontal: 20.0),
        child: Center(
          child: Column(
            mainAxisSize: MainAxisSize.min,
            children: [
              TextButton(
                onPressed: () {
                  VRouter.of(context).to(
                    context.vRouter.url,
                    isReplacement: true,
                    historyState: {'count': '${count + 1}'},
                  );
                  setState(() => count++);
                },
                child: Container(
                  decoration: BoxDecoration(
                    borderRadius: BorderRadius.circular(50),
                    color: Colors.blueAccent,
                  ),
                  padding:
                      EdgeInsets.symmetric(horizontal: 20.0, vertical: 8.0),
                  child: Text(
                    'Your pressed this button $count times',
                    style: buttonTextStyle,
                  ),
                ),
              ),
              SizedBox(height: 20),
              Text(
                'This number is saved in the history state so if you are on the web leave this page and hit the back button to see this number restored!',
                style: textStyle,
                textAlign: TextAlign.center,
              ),
            ],
          ),
        ),
      ),
    );
  }

  void getCountFromState(BuildContext context) {
    setState(() {
      count = (VRouter.of(context).historyState['count'] == null)
          ? 0
          : int.tryParse(VRouter.of(context).historyState['count'] ?? '') ?? 0;
    });
  }
}

class SettingsWidget extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Padding(
      padding: const EdgeInsets.symmetric(horizontal: 20.0),
      child: Center(
        child: Column(
          mainAxisSize: MainAxisSize.min,
          children: [
            Text(
              'Did you see the custom animation when coming here?',
              style: textStyle.copyWith(fontSize: textStyle.fontSize! + 2),
            ),
          ],
        ),
      ),
    );
  }
}

final textStyle = TextStyle(color: Colors.black, fontSize: 16);
final buttonTextStyle = textStyle.copyWith(color: Colors.white);
```

</details>

Sorry for this, the issue is that I can't reproduce it and I have really no idea why this might happen (since logs are just `print`s)

**Comment by yk-sgr** [comment]
The same issue.

![](https://i.imgur.com/GPLxXLc.gif)

Reload is not finishing.

**Comment by yk-sgr** [comment]
```
I/flutter ( 8883): [VRouter: INFO] Stopped the navigation to / which used VRouter.to o
I/flutter ( 8883): [VRouter: INFO] Successfully navigated to "/login" using VRouter.to o
[{"id":0,"result":{"value":"android","type":"_extensionType","method":"ext.flutter.platformOverride"}}]
[{"id":1,"result":{"timeDilation":"1.0","type":"_extensionType","method":"ext.flutter.timeDilation"}}]
[{"id":2,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaint"}}]
[{"id":3,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaintBaselinesEnabled"}}]
[{"id":4,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.repaintRainbow"}}]
[{"id":5,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.showPerformanceOverlay"}}]
[{"id":6,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.debugAllowBanner"}}]
[{"id":7,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":8,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.show"}}]
[{"id":9,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":10,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
[{"id":11,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":12,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":13,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
[{"event":"app.progress","params":{"appId":"d3f9c01b-e633-442d-8d3e-1f07206b9092","id":"2","progressId":"hot.restart","message":"Performing hot restart...","finished":false}}]
[{"event":"app.progress","params":{"appId":"d3f9c01b-e633-442d-8d3e-1f07206b9092","id":"3","progressId":null,"message":"Syncing files to device Pixel 4a...","finished":false}}]
[{"event":"app.progress","params":{"appId":"d3f9c01b-e633-442d-8d3e-1f07206b9092","id":"3","progressId":null,"finished":true}}]
[{"event":"app.progress","params":{"appId":"d3f9c01b-e633-442d-8d3e-1f07206b9092","id":"2","progressId":"hot.restart","finished":true}}]
Restarted application in 927ms.
[{"id":14,"result":{"code":0,"message":""}}]
I/flutter ( 8883): [VRouter: INFO] Stopped the navigation to / which used VRouter.to o
I/flutter ( 8883): [VRouter: INFO] Successfully navigated to "/login" using VRouter.to o
[{"id":15,"result":{"value":"android","type":"_extensionType","method":"ext.flutter.platformOverride"}}]
[{"id":16,"result":{"timeDilation":"1.0","type":"_extensionType","method":"ext.flutter.timeDilation"}}]
[{"id":17,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaint"}}]
[{"id":18,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.debugPaintBaselinesEnabled"}}]
[{"id":19,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.repaintRainbow"}}]
[{"id":20,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.showPerformanceOverlay"}}]
[{"id":21,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.debugAllowBanner"}}]
[{"id":22,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":23,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":24,"result":{"enabled":"true","type":"_extensionType","method":"ext.flutter.inspector.structuredErrors"}}]
[{"id":25,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.show"}}]
[{"id":26,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":27,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
[{"id":28,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRebuildDirtyWidgets"}}]
[{"id":29,"result":{"enabled":"false","type":"_extensionType","method":"ext.flutter.inspector.trackRepaintWidgets"}}]
```

**Comment by lulupointu** [comment]
Do you **hot restart** or **hot reload** ?

**Comment by yk-sgr** [comment]
In the Gif I used hot restart. But it's both not available anymore.

**Comment by rebaz94** [comment]
> For which platform are you developing ?
> 
> @rebaz94 can you also answer this one ?

I am developing for Mac & Web.

This problem only happen when you run from IDE and the first time hot reload works and when you update after of that hot reload button is disable but if you use terminal does not have any problem

**Comment by rebaz94** [comment]
@lulupointu I think I found why this problem happen, it's not just for ```info``` logs but also happen for ```warning```.
But the ```warning logs``` does not happen so frequently, its why not problem for us.

But I removed the log ```color``` from ```v_log_printer.dart``` and everything work as expected
```dart
static const Map<VLogLevel, String> _vLogLevelsColor = {
    VLogLevel.info: '', // the color cause the problem
    VLogLevel.warning: '', // Yellow
    // VLogLevel.error: '',
  };
```

**Comment by golane-august** [comment]
I think this started happening when using the new flutter `2.8.0`

**Comment by sperochon** [comment]
Hello,

Had the same issue and works fine when removing color as suggests by @rebaz94 👍 (Thanks!)

Flutter 2.8.1

**Comment by jasonJamEther** [comment]
I am having the same issue. Where is v_log_printer.dart located?

**Comment by doldremus** [comment]
Hello
Same with Flutter 2.8.1 (stable channel) and  2.10.0-0.3.pre (dev channel), Android Studio 2021.1.1

**Comment by lulupointu** [comment]
This should be fixed thanks to @doldremus ! (In v1.2.0+19)

Thanks again since it was hard for me to fix, not being able to replicate this.

Let me know if someone still has an issue.
